.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,70 @@
+name: Bug report
+description: Create a bug report to help us improve. Use this template if you encountered an issue while integrating with or implementing the APIs of this SDK.
+labels: [bug, triage-required]

```

Comment:
Once these changes are approved, I can create all the labels required for these templates.